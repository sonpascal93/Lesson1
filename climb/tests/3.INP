50
5 -5 7 0 -3 7 -4 -1 -2 0 -7 -1 5 -3 0 -2 -6 0 -2 6 -9 5 -9 4 9 6 0 5 0 7 -1 8 3 7 -6 0 -9 0 9 5 -3 -4 4 -9 7 -5 -6 1 -9 -4 