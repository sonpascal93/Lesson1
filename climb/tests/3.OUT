18
2 7 9 12 15 18 20 22 24 27 29 36 38 41 42 46 47 49 